frontend/src/scenes/persons/personsLogic.test.ts

```diff
@@ -0,0 +1,59 @@
+import { defaultAPIMocks, mockAPI } from 'lib/api.mock'
+import { expectLogic } from 'kea-test-utils'
+import { initKeaTestLogic } from '~/test/init'
+import { personsLogic } from './personsLogic'
+import { router } from 'kea-router'
+
+jest.mock('lib/api')
+
+describe('personsLogic', () => {
```

Comment:
nit: should we also test for filters with a value? i.e. not `is_set` only?

Reply:
all this test is testing that it's correctly passing around the properties between the logic and the url, so the value shouldn't matter.